### PR TITLE
Add replication to test-artifacts.k8s.io terraform

### PR DESCRIPTION
**Pull request by justinsb**
Builds on #4460 , adding replication to two regions, just to give a more realistic test environment and so we can reuse this configuration in prod with more regions.

- infra/aws: create S3 buckets for artifacts.k8s.io
- test-artifacts.k8s.io terraform: Add replication configuration

### CODE REVIEW

**Comment by justinsb** [comment]
/hold

I don't think we want to use AWS replication here, let's see how it goes directly copying to each of our mirrors.

File: infra/aws/terraform/artifacts.k8s.io/main.tf [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```
Comment (TerryHowe) [inline infra/aws/terraform/artifacts.k8s.io/main.tf:2]:
```suggestion
Copyright The Kubernetes Authors.
```
Year is old. Not sure if year from copyright notices are allowed here. It saves a lot of hassle.

**Comment by justinsb** [comment]
We are not currently using AWS replication and it seems to be working well (at least when I set up IAM correctly :-) ), so going to close this; we can always reopen if we change our mind.

/close

**Pull request closed without merging**